tests/setupTests.jsx

```
@@ -1,56 +1,69 @@
-  setItem: jest.fn(),
-  removeItem: jest.fn(),
-  clear: jest.fn(),
+  getItem: jest.fn((key) => store[key] ?? null),
+  setItem: jest.fn((key, value) => {
+    store[key] = String(value);
+  }),
+  removeItem: jest.fn((key) => {
+    delete store[key];
+  }),
+  clear: jest.fn(() => {
+    Object.keys(store).forEach((key) => delete store[key]);
```

Biome (2.4.11)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)